tests/network/test_reaction_network.py

```diff
@@ -357,57 +333,90 @@ def test_solve_prerequisites(self):
         for entry in RN.entries["C3 H4 O3"][10][0]:
             if self.EC_mg.isomorphic_to(entry.mol_graph):
                 EC_ind = entry.parameters["ind"]
-                break
+            break
         Li1_ind = RN.entries["Li1"][0][1][0].parameters["ind"]
 
         # perfrom calc
-        PRs_calc, old_solved_PRs = RN.solve_prerequisites(
-            [EC_ind, Li1_ind], weight="softplus"
-        )
+        PRs_calc, old_solved_PRs = RN.solve_prerequisites([EC_ind, Li1_ind], weight="softplus")
 
         # assert
-        PR_paths = copy.deepcopy(RN_loaded_pr_solved.PRs)
-
-        for node in PRs_calc:
-            for start in PRs_calc[node]:
-                self.assertEqual(
-                    [
-                        PRs_calc[node][start].all_prereqs,
-                        PRs_calc[node][start].byproducts,
-                        PRs_calc[node][start].full_path,
-                        PRs_calc[node][start].path,
-                        PRs_calc[node][start].solved_prereqs,
+        with open(os.path.join(test_dir, "unittest_RN_pr_solved_PRs.pkl"), "rb") as input:
+            PR_paths = pickle.load(input)
+        print(PRs_calc[12])
+        print(PR_paths[12])
+
+        for node in PR_paths:
+            for start in PR_paths[node]:
+                try:
+                    self.assertEqual(
+                        PRs_calc[node][start].all_prereqs, PR_paths[node][start].all_prereqs,
+                    )
+                    self.assertEqual(
+                        PRs_calc[node][start].byproducts, PR_paths[node][start].byproducts,
+                    )
+                    self.assertEqual(
+                        PRs_calc[node][start].solved_prereqs, PR_paths[node][start].solved_prereqs,
+                    )
+                    self.assertEqual(
                         PRs_calc[node][start].unsolved_prereqs,
-                    ],
-                    [
-                        PR_paths[node][start].all_prereqs,
-                        PR_paths[node][start].byproducts,
-                        PR_paths[node][start].full_path,
-                        PR_paths[node][start].path,
-                        PR_paths[node][start].solved_prereqs,
                         PR_paths[node][start].unsolved_prereqs,
-                    ],
-                )
-
-                if PRs_calc[node][start].cost != PR_paths[node][start].cost:
-                    self.assertAlmostEqual(
-                        PRs_calc[node][start].cost, PR_paths[node][start].cost, places=2
-                    )
-                if PRs_calc[node][start].pure_cost != PR_paths[node][start].pure_cost:
-                    self.assertAlmostEqual(
-                        PRs_calc[node][start].pure_cost,
-                        PR_paths[node][start].pure_cost,
-                        places=2,
                     )
+                    for i in range(
```

Comment:
This chunk could use some comments. I mostly get what's happening here, but a year in the future it's possible I'd find it totally incomprehensible